pyfolio/plotting.py

```diff
@@ -29,6 +29,8 @@
 from . import timeseries
 from . import pos
 
+from .utils import BDAYS_PER_MONTH
```

Comment:
@CaptainKanuk I like this timedelta approach, but I will let @twiecki weigh in. I'm just not sure if forcing a timedelta type of contruct (?) or pandas dependency may hamper potential usecases down the road that aren't pandas-friendly
ie: as we build out pyfolio to include additional analytical aspects, etc